tests/index.spec.js

```diff
@@ -392,6 +392,53 @@ describe('Testing generic actions', () => {
     expect(result.fee_collected).to.be.a('number')
   })
 
+  it(`RE Add public addresses`, async () => {
+    const result = await fioSdk.genericAction('addPublicAddresses', {
```

Comment:
I think we can move this part to the `Remove all public addresses` section (without expect statements), above `const result = await fioSdk.genericAction('removeAllPublicAddresses', {`

Reply:
ill move it...